internal/server/server.go

```diff
@@ -140,56 +135,77 @@ func (s *Server) newRouter() *mux.Router {
 
 	router.Handle("/schema", stdMiddleware(http.HandlerFunc(s.schemaHandler)))
 
+	router.Handle("/photo/{team}", stdMiddleware(http.HandlerFunc(s.photoHandler)))
+
 	router.Handle("/analysis/{eventKey}", cors(http.HandlerFunc(s.eventAnalysisHandler), []string{"GET"}))
 	router.Handle("/analysis/{eventKey}/{team}", cors(http.HandlerFunc(s.teamAnalysisHandler), []string{"GET"}))
 	router.Handle("/analysis/{eventKey}/{matchKey}/{color}", cors(http.HandlerFunc(s.allianceAnalysisHandler), []string{"GET"}))
 
 	return router
 }
 
-func cors(next http.Handler, allowedMethods []string) http.Handler {
-	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		w.Header().Set("Access-Control-Allow-Origin", "*")
-		w.Header().Set("Access-Control-Allow-Methods", strings.Join(append(allowedMethods, "OPTIONS"), ","))
+const imgurFormat = "http://i.imgur.com/%sl.jpg"
```

Comment:
Ok I don't really know anything about Golang constants, would it be better to stick this in utils or would it just be confusing to have it there?

Reply:
It is used in the function below, it makes sense to have it right above.